src/schema/workflow_execution_activity.yaml

```diff
@@ -275,6 +276,10 @@ classes:
         structured_pattern:
           syntax: "{id_nmdc_prefix}:(omprc|dgms)-{id_shoulder}-{id_blade}$"
           interpolated: true
+      uses_calibration:
+        structured_pattern:
+          syntax: "{id_nmdc_prefix}:calib-{id_shoulder}-{id_blade}$"
+          interpolated: true
```

Comment:
Same comment as above - move the structured_pattern to the global definition of the slot so its in same place as the definition of the range.